### PR TITLE
Use jitsi-utils logger for logging, rather than System.out and printf 

### PR DESCRIPTION
**Pull request by JonathanLennox**

### CODE REVIEW

File: jniwrapper/native/src/org_jitsi_modified_sctp4j_SctpJni.c [hunk 1]
```diff
@@ -425,14 +437,41 @@ JNI_OnLoad(JavaVM *vm, void *reserved)
...
+
+                    if (logDebugCb && logInfoCb && logErrorCb)
                     {
-                        Sctp_clazz = clazz;
-                        Sctp_receiveCb = receiveCb;
-                        Sctp_sendCb = sendCb;
-                        Sctp_vm = vm;
-                        r = JNI_VERSION_1_4;
+                        clazz = (*env)->NewGlobalRef(env, clazz);
+                        if (clazz)
```
Comment (bgrozev) [inline jniwrapper/native/src/org_jitsi_modified_sctp4j_SctpJni.c:464]: Why the nested `if`s? Can't we just set the different callbacks independently (I think the rest of the code checks each one before they are used)? Should we print something if one of them is not available?

File: jniwrapper/java/src/main/java/org/jitsi_modified/sctp4j/SctpJni.java [hunk 2]
```diff
@@ -168,4 +172,31 @@ public static int onSctpOutboundPacket(
         // FIXME handle tos and set_df
         return -1;
     }
+
+    /**
+     * Method fired by native counterpart to log a debug message.
+     * @param message the message to log
+     */
+    public static void logDebug(String message) {
+        message = message.trim();
```
Comment (bgrozev) [inline jniwrapper/java/src/main/java/org/jitsi_modified/sctp4j/SctpJni.java:181]: The trim() is just to remove the "\n" that the native code adds? Can we just remove the "\n" from the native code?

File: jniwrapper/native/src/org_jitsi_modified_sctp4j_SctpJni.c (hunk 1 above)
Comment (JonathanLennox) [reply jniwrapper/native/src/org_jitsi_modified_sctp4j_SctpJni.c:464]: I was just copying what the existing code was doing.

File: jniwrapper/java/src/main/java/org/jitsi_modified/sctp4j/SctpJni.java (hunk 2 above)
Comment (JonathanLennox) [reply jniwrapper/java/src/main/java/org/jitsi_modified/sctp4j/SctpJni.java:181]: At least for logError, I have a fallback to native (v)printf if we can't successfully call the Java method, so you can get meaningful error messages for failure to load the native classes.

Also the debug function is passed as a native callback to the usrsctp library, and I assume that has newlines.

**Pull request merged**